krakentools/d4a2fbe/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:jammy as app
+
+WORKDIR /
+
+# using a git commit because the last release was over three years ago,, despite recent(ish) activity in the repo's main branch
+ARG KRAKENTOOLS_VERSION="d4a2fbe2833c2392a724697c0c97a3014e748474"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="KrakenTools"
+LABEL software.version=${KRAKENTOOLS_VERSION}
+LABEL description="KrakenTools is a set of tools for processing Kraken output"
+LABEL website="https://github.com/jenniferlu717/KrakenTools"
+LABEL documentation="https://github.com/jenniferlu717/KrakenTools"
+LABEL license="https://github.com/jenniferlu717/KrakenTools/blob/master/LICENSE"
+LABEL maintainer="Sage Wright"
+LABEL maintainer.email="sagemwright@gmail.com"
+
+
+# Install dependencies via apt-get; cleanup apt garbage
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget \
+    ca-certificates \
+    python3 \
+    python3-pip \
+    python3-setuptools \
+    gawk \
+    jq \
+    git \
+    procps && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+RUN pip install biopython \
+    pandas 
```

Comment:
looks like when the github actions ran, it install biopython 1.84 and pandas 2.2.3.

I would recommend pinning these versions as I've seen these packages upgrade and break things. Can we pin these with `pip install biopython==1.84 pandas==2.2.3` ?

That is assuming these versions are functional with the krakentools scripts